package.json

```diff
@@ -37,9 +37,7 @@
     "babel-preset-stage-0": "^6.0.15",
     "bootstrap-loader": "^2.0.0-beta.21",
     "bootstrap-sass": "^3.3.7",
-    "butter-base-theme": "^0.1.2",
-    "butter-theme-dark": "^0.1.3",
-    "butter-theme-pink": "^0.0.1",
+    "butter-installer-theme": "^0.1.0",
```

Comment:
it's a pack, anyway as we discussed on slack please remove it and load themes that are in package.json